### ISSUE TITLE
No any templates in Ignore File Generator Window

### ISSUE THREAD

**Issue by ouyangjunfei** [JetBrains/idea-gitignore#648]
### Prerequisites

* [x] Plugin is in the latest version

* [x] Issue was not reported yet
* [ ] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

**I don't know where to find Stack trace log, if possible please tell me!**

The plugin works fine in last IDEA version, and I update my IDEA some days ago.

I know this Plugin **CANNOT** work in latest IDEA 2020.3 by *ISSUES*, so I wait the newer VERSION.

But when I upgrade to VERSION 4.0.0, there are **NO** templates.

### Steps to Reproduce

1. Right click top module to add a *.gitignore*
2. New -> .ignore File -> .gitignore File 
3. Ignore File Generator Windows pops up but there are none templates

**Expected behavior:**

Many templates about languages or frameworks to choose

**Actual behavior:** 

![image](https://user-images.githubusercontent.com/24387167/101969821-8b360680-3c61-11eb-9654-f32ad8a04e37.png)

**Reproduces how often:** 

Just happened after I upgraded to .ignore Plugin Version 4.0.0

### Versions

**Plugin:** 

![image](https://user-images.githubusercontent.com/24387167/101969569-e535cc80-3c5f-11eb-97a1-0e5d1cd041bd.png)

```
4.0.0
```

**IDE:** 

```
IntelliJ IDEA 2020.3 (Ultimate Edition)
Build #IU-203.5981.155, built on December 1, 2020
Licensed to XXXXXXX
Subscription is active until April 2, 2021
For educational use only.
Runtime version: 11.0.9+11-b1145.21 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: ide.balloon.shadow.size=0
Non-Bundled Plugins: com.intellij.ideolog, com.jetbrains.plugins.ini4idea, com.markskelton.one-dark-theme, com.github.setial, com.intellij.struts2, mobi.hsz.idea.gitignore
```

**OS:** 

```
Windows 10 Home 1909
```

**Comment by hsz** [comment]
Thanks. It appeared that it's a CI issue - [github/gitigore](https://github.com/github/gitignore) submodule was not fetched when releasing the final artifact to the Marketplace.
It's fixed now and will be published with `v4.0.1` soon.
Sorry for the inconvenience!

**Comment by ouyangjunfei** [comment]
@hsz After I upgrade the plugin to VERSION 4.0.1, functions are all good! THANKS!
